airbyte-integrations/connectors/source-facebook-marketing/acceptance-test-config.yml

```diff
@@ -13,21 +13,10 @@ tests:
     - config_path: "secrets/config.json"
   basic_read:
     - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/configured_catalog.json"
-      timeout_seconds: 600
+      empty_streams: ["videos"]
   incremental:
     - config_path: "secrets/config.json"
       configured_catalog_path: "integration_tests/configured_catalog_without_insights.json"
       future_state_path: "integration_tests/future_state.json"
   full_refresh:
     - config_path: "secrets/config.json"
```

Comment:
means we test all streams from the catalog, and because we using new PK-based comparison we don't need to ignore some fields